### PR TITLE
chore: add OpenShift info

### PR DESCRIPTION
**Pull request by rafasdc**
Implements #1271

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -184,3 +191,85 @@ Required dependencies:
...
+
+#### Certbot
+
+As the name implies; a job that uses certbot to keep the TLS certificate up to date. Runs everyday at 5:00PM Pacific Time.
+
+#### Receive applications
+
+Marks all applications for a specific intake as received on the database. Runs twice a day at 10:00 AM and 10:00PM Pacific time.
+
+ANYTHING ELSE HERE?
```
Comment (marcellmueller) [inline README.md:259]: I'd just add that receive applications changes all applications with status `submitted` to `received`.

File: README.md [hunk 2]
```diff
@@ -184,3 +191,85 @@ Required dependencies:
...
+
+3. Create any updated values as needed for your new deployment under `helm/app`. For example, if you named your environment `foo` you will create a file named `values-foo.yaml`
+4. Add an extra step to `.github/workflows/deploy.yaml` with updated job and environment name.
+5. Run the action!
+
+Note: there might be additional modifications or steps required to suit your specific needs, You might need to create independent workflows or Helm charts.
+
+### Disaster recovery documentation
+
+TBD
```
Comment (marcellmueller) [inline README.md:237]: https://github.com/bcgov/CONN-CCBC-portal/blob/main/docs/Disaster_recovery_Patroni.md

Not sure why this isn't in the main `README.md` table of contents, I might have forgotten it 😅

File: README.md [hunk 3]
```diff
@@ -184,3 +191,85 @@ Required dependencies:
...
+
+## OpenShift information
+
+### General information
+
+- [The Private Cloud as a Service Platform Technical Documentation](https://docs.developer.gov.bc.ca/)
+
+### Deploying the project
+
+To deploy the project into a a new namespace or to deploy another instance of the project into an existing namespace GitHub Environments along with Helm and GitHub Actions is used. The following steps can be used to as a reference to deploy:
```
Comment (marcellmueller) [inline README.md:203]: I believe we need to first manually deploy with Helm to generate our deployer service account, then we can add the environment variables and deploy through GitHub. 

There is a default `deployer` service account set up in the namespaces but I am pretty sure it doesn't have all the permissions needed which is why we generate our own eg `ccbc-deployer`.

File: README.md (hunk 2 above)
Comment (rafasdc) [reply README.md:237]: Thanks! Added :)

File: README.md (hunk 3 above)
Comment (rafasdc) [reply README.md:203]: Added some info on that.

**Review by VB-at-Bis — APPROVED** [review]
Perfect

**Pull request merged**